lib/PostMessageTransport.js

```diff
@@ -58,6 +61,15 @@ define(function (require, exports, module) {
     function send(idOrArray, msgStr){
         var win = _iframeRef.contentWindow;
 
+        // Parse msgStr and check to see if it's a reload command.
+        // If so, trigger launcher.launch and return
+        var msg = JSON.parse(msgStr);
+
+        if(msg.method === "Page.reload" || msg.method === "Page.navigate") {
```

Comment:
Playing with this patch locally, I'm finding it odd to have the reload happen when I navigate to another file in the editor.  When I go to a CSS file, suddenly it loads the CSS file into the preview, which is probably not what we want.